src/Tizen.MachineLearning.Inference/Tizen.MachineLearning.Inference/MlInformation.cs

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+
+namespace Tizen.MachineLearning.Inference
+{
+    /// <summary>
+    /// The MlInformation class manages information based on key-value.
+    /// </summary>
+    /// <since_tizen> 9 </since_tizen>
```

Comment:
plz check version (latest version of c# api would be 12)

Reply:
Updated!
Thank you